examples/subscribe_invoices.rs

```diff
@@ -0,0 +1,32 @@
+// This program accepts three arguments: address, cert file, macaroon file
+// The address must start with `https://`!
+
+#[tokio::main]
+async fn main() {
+    let mut args = std::env::args_os();
+    args.next().expect("not even zeroth arg given");
+    let address = args.next().expect("missing arguments: address, cert file, macaroon file");
+    let cert_file = args.next().expect("missing arguments: cert file, macaroon file");
+    let macaroon_file = args.next().expect("missing argument: macaroon file");
+    let address = address.into_string().expect("address is not UTF-8");
+
+    // Connecting to LND requires only address, cert file, and macaroon file
+    let mut client = tonic_lnd::connect(address, cert_file, macaroon_file)
+        .await
+        .expect("failed to connect");
+
+    let mut invoice_stream = client
+        .subscribe_invoices(tonic_lnd::rpc::InvoiceSubscription { add_index: 0, settle_index: 0 })
+        .await
+        .expect("Failed to call subscribe_invoices")
+        .into_inner();
+
+    while let Some(invoice) = invoice_stream.message().await.expect("Failed to receive invoices") {
+        if let Some(state) = tonic_lnd::rpc::invoice::InvoiceState::from_i32(invoice.state) {
+            // If this invoice was Settled we can do something with it
+            if state == tonic_lnd::rpc::invoice::InvoiceState::Settled {
+                println!("{invoice:?}");
```

Comment:
Oh, just noticed this doesn't obey MSRV. I think it's fine for an example, just be mindful if you want to contribute into the library next time.

Reply:
what do you mean with `doesn't obey MSRV`? what part of the code? it works I tested and it works

Reply:
Not writing the arguments in `println` was only stabilized recently but this crate supports Rust 1.48.0